packages/accounts/src/account.ts

```diff
@@ -711,4 +727,29 @@ export class Account {
             total: summary.total.toString(),
         };
     }
+
+    /**
+     * @param message The message that wants to be transmitted.
+     * @param recipient The recipient to whom the message is destined (e.g. "alice.near" or "myapp.com").
+     * @param nonce A nonce that uniquely identifies this instance of the message, denoted as a 32 bytes array (a fixed `Buffer` in JS/TS).
+     * @param callbackUrl Optional, applicable to browser wallets (e.g. MyNearWallet). The URL to call after the signing process. Defaults to `window.location.href`.
+     */
+    async signMessage({ message, recipient, nonce, callbackUrl }: SignMessageParams): Promise<SignedMessage> {
+        // Check the nonce is a 32bytes array
+        if (nonce.length != 32) { throw Error('Expected nonce to be a 32 bytes buffer'); }
+        
+        const tag = 2147484061;
```

Comment:
 This looks to me like a magic number. Please give it some context or extract it to a constant somewhere

Reply:
will extract it to a constant